### PR TITLE
[DSW-276] Notification when mailer fails to send email

### PR DESCRIPTION
**Pull request by MarekSuchanek**

### CODE REVIEW

File: lib/Service/Mail/Mailer.hs [hunk 1]
```diff
@@ -137,8 +137,8 @@ makeConnection True host (Just port) = SMTPSSL.doSMTPSSLWithSettings host settin
   where
     settings = SMTPSSL.defaultSettingsSMTPSSL {SMTPSSL.sslPort = fromIntegral port}
 
-sendEmail :: [String] -> TL.Text -> [MIME.Part] -> AppContextM ()
-sendEmail to subject [] = return () -- empty mail won't be sent
+sendEmail :: [String] -> TL.Text -> [MIME.Part] -> AppContextM (Either String ())
+sendEmail to subject [] = return $ Left "Tried to sent empty email" -- empty mail won't be sent
```
Comment (vknaisl) [inline lib/Service/Mail/Mailer.hs]: Please put it to the `Localization` file

File: lib/Service/Mail/Mailer.hs [hunk 2]
```diff
@@ -111,15 +111,15 @@ loadMailTemplateParts mailName context = do
     then do
       templateFileParts <- loadFileParts $ root ++ _MAIL_TEMPLATE_ATTACHMENTS_FOLDER
       globalFileParts <- loadFileParts $ commonRoot ++ _MAIL_TEMPLATE_ATTACHMENTS_FOLDER
-      return $ mainParts ++ templateFileParts ++ globalFileParts
-    else return []
+      return . Right $ mainParts ++ fromRight [] templateFileParts ++ fromRight [] globalFileParts
+    else return $ Left "No HTML neither PLAIN TEXT"
```
Comment (vknaisl) [inline lib/Service/Mail/Mailer.hs]: Please put it to the Localization file

**Review by vknaisl — CHANGES_REQUESTED** [review]

File: lib/Service/Mail/Mailer.hs (hunk 1 above)
Comment (MarekSuchanek) [reply lib/Service/Mail/Mailer.hs]: Fixed in https://github.com/ds-wizard/dsw-server/pull/56/commits/9138cc713207b11a3590d895176d5d1558b8b19d

File: lib/Service/Mail/Mailer.hs (hunk 2 above)
Comment (MarekSuchanek) [reply lib/Service/Mail/Mailer.hs]: Fixed in https://github.com/ds-wizard/dsw-server/pull/56/commits/9138cc713207b11a3590d895176d5d1558b8b19d

**Review by vknaisl — APPROVED** [review]

**Pull request merged**